exporter/otelarrowexporter/factory.go

```diff
@@ -97,13 +97,13 @@ func createTracesExporter(
 	set exporter.CreateSettings,
 	cfg component.Config,
 ) (exporter.Traces, error) {
-	exp, err := newExporter(cfg, set, createArrowTracesStream)
+	oce, err := newExporter(cfg, set, createArrowTracesStream)
```

Comment:
Not clear why this variable was renamed - what is `oce` stand for? :thinking:

Reply:
The OTLP-general stuff up two levels was derived from the core `otlpexporter` and still has some irregular names as a result. I've tried to fix them as a I notice them. I fixed them in the skeleton for this component, but copied the old names by mistake when adding code from the `otel-arrow` repo. Will fix!